tests/process_monitor/test_comm_from_server.py

```diff
@@ -664,16 +664,40 @@ def test_MantarrayProcessesMonitor__check_and_handle_server_to_main_queue__handl
     monitor_thread, *_ = test_monitor(test_process_manager)
     server_to_main_queue = test_process_manager.queue_container.from_flask
 
-    okc_process = test_process_manager.instrument_comm_process
     fw_process = test_process_manager.file_writer_process
     da_process = test_process_manager.data_analyzer_process
-    expected_okc_item = "item 1"
-    expected_fw_item = "item 2"
-    expected_da_item = "item 3"
+    expected_okc_item = {
+        "board_0": {"file_writer_to_data_analyzer": [], "outgoing_data": []},
```

Comment:
OkComm does not have a `file_writer_to_data_analyzer` queue, Only FileWriter and DataAnalyzer would have that

Reply:
We also want to make sure there are items in these lists